SPECS/php/php.spec

```diff
@@ -10,22 +9,14 @@
 # Please preserve changelog entries
 #
 # API/ABI check
-
 %global apiver      20190902
 %global zendver     20190902
 %global pdover      20170320
-
 # Adds -z now to the linker flags
-%global _hardened_build 1
-
 # version used for php embedded library soname
 %global embed_version 7.4
```

Comment:
Would love to see this macro be derived from the version macro- you can search for `%define majmin` in this repo to see examples of how we've done it in the past.